yarn.lock

```diff
@@ -10,6 +10,185 @@
     "@jridgewell/gen-mapping" "^0.1.0"
     "@jridgewell/trace-mapping" "^0.3.9"
 
+"@appium/base-driver@^9.0.0", "@appium/base-driver@^9.1.0", "@appium/base-driver@^9.3.10":
```

Comment:
> as you quickly should notice, this new folder that is being added packages/rn-tester-e2e at the moment is not part of the yarn workspace because the dependencies it pulls in are quite heavily, so we don't want to force everyone to download them every time. We will have to revisit this once we wire it up on CI

Why are we seeing the dependencies in the root lockfile. Is this change stale?

Reply:
@kelset WDYT?

Reply:
update on this: we'll actually change approach and add a new CircleCI job that runs this E2E in this PR already.